proto/cmp/types/v1alpha/currency.proto

```diff
@@ -2,170 +2,220 @@ syntax = "proto3";
 
 package cmp.types.v1alpha;
 
+import "google/protobuf/empty.proto";
+import "cmp/types/v1alpha/token.proto";
+
+// Definition of the price message:
+// The combination of "value" and "decimals" fields express always the value of the
+// currency, not of the fraction of the currency [ ETH not wei, CAM and not aCAM,
+// BTC and not Satoshi, EUR not EUR-Cents ]
+// Be aware that partners should not do rounding with crypto currencies.
+//
+// Example implementations:
+// off-chain payment of 100 € or 100 $:
+//  value=10000
+//  decimals=2
+//  iso_currency=EUR or USD
+//
+// on-chain payment of 100.65 EURSH
+//  value=10065
+//  decimals=2
+//  token_currency=EURSH
+//  contract_address=0x5b1c852dad36854B0dFFF61d2C13F108D8E01975
+//  this currency has 5 decimals on Columbus and conclusively to mint the value of
```

Comment:
These will not be used for minting any tokens. So, we should not use "mint" here to avoid confusion. See the comment below with suggested change.

Reply:
lets discuss, "to mint the value of 0.0065" - this refers to a transaction, maybe another word for registering a transaction with a value of xxx should be used

Reply:
use "create transaction" in these examples

Reply:
done